<launch>
	<node name="tf_to_pose" pkg="ros_conversions" type="tf_to_pose" output="screen">
			<remap from="pose_topic" to="pose"/>
			
			<param name="tf_target_frame" value="map"/>
			<param name="tf_source_frame" value="base_link"/>
			<param name="pose_frame_id" value="map"/>
			
			<param name="rate" type="double" value="50.0"/>
		</node>
</launch>
